packages/admin/admin/src/tabs/Tabs.tsx

```diff
@@ -1,39 +1,34 @@
 import { ComponentsOverrides, Theme } from "@mui/material";
-import MuiTab, { TabProps as MuiTabProps } from "@mui/material/Tab";
 import MuiTabs, { TabsProps as MuiTabsProps } from "@mui/material/Tabs";
 import { WithStyles, withStyles } from "@mui/styles";
 import * as React from "react";
 
+import { CustomDivider, Divider, DividerProps } from "./CustomDivider";
+import { CustomTab, Tab, TabProps } from "./Tab";
 import { styles, TabsClassKey } from "./Tabs.styles";
 import { TabScrollButton } from "./TabScrollButton";
 
-interface TabProps extends Omit<MuiTabProps, "children"> {
-    label: React.ReactNode;
-    forceRender?: boolean;
-    children: React.ReactNode;
-}
-
-export const Tab: React.SFC<TabProps> = () => null;
-
 interface ITabsState {
     value: number;
     setValue: (value: number) => void;
 }
 
 export interface TabsProps extends MuiTabsProps {
-    children: Array<React.ReactElement<TabProps> | boolean | null | undefined> | React.ReactElement<TabProps>;
-    tabComponent?: React.ComponentType<MuiTabProps>;
+    children: Array<React.ReactElement<TabProps | DividerProps> | boolean | null | undefined>;
+    tabComponent?: React.ComponentType<TabProps>;
```

Comment:
Could this change from `MuiTabProps` to `TabProps` cause any compatibility problems?

Reply:
The only thing would be `icon` and `tabIcon` I guess. Since @johnnyomair also stated this aswell, I changed it back to `icon`.